cli/restore.go

```diff
@@ -369,37 +369,39 @@ type descrRestoreOpts struct {
-	Backup             string           `yaml:"backup" json:"backup"`
-	RestoreFrom        *string          `yaml:"restore_from,omitempty" json:"restore_from,omitempty"`
-	RestoreTo          *string          `yaml:"restore_to,omitempty" json:"restore_to,omitempty"`
-	Type               pbm.BackupType   `yaml:"type" json:"type"`
-	Status             pbm.Status       `yaml:"status" json:"status"`
-	Error              *string          `yaml:"error,omitempty" json:"error,omitempty"`
-	Namespaces         []string         `yaml:"namespaces,omitempty" json:"namespaces,omitempty"`
-	Replsets           []RestoreReplset `yaml:"replsets" json:"replsets"`
-	OPID               string           `yaml:"opid" json:"opid"`
-	StartTS            int64            `yaml:"-" json:"start_ts"`
-	StartTime          string           `yaml:"start" json:"-"`
-	LastTransitionTS   int64            `yaml:"-" json:"last_transition_ts"`
-	LastTransitionTime string           `yaml:"last_transition_time" json:"-"`
+	Name               string           `json:"name" yaml:"name"`
+	OPID               string           `json:"opid" yaml:"opid"`
+	Backup             string           `json:"backup" yaml:"backup"`
+	Type               pbm.BackupType   `json:"type" yaml:"type"`
+	Status             pbm.Status       `json:"status" yaml:"status"`
+	Error              *string          `json:"error,omitempty" yaml:"error,omitempty"`
+	Namespaces         []string         `json:"namespaces,omitempty" yaml:"namespaces,omitempty"`
+	StartTS            *int64           `json:"start_ts,omitempty" yaml:"-"`
+	StartTime          *string          `json:"start,omitempty" yaml:"start,omitempty"`
+	PITR               *int64           `json:"ts_to_restore,omitempty" yaml:"-"`
+	PITRTime           *string          `json:"time_to_restore,omitempty" yaml:"time_to_restore,omitempty"`
+	LastTransitionTS   int64            `json:"last_transition_ts" yaml:"-"`
+	LastTransitionTime string           `json:"last_transition_time" yaml:"last_transition_time"`
+	Replsets           []RestoreReplset `json:"replsets" yaml:"replsets"`
 }
 
 type RestoreReplset struct {
-	Name               string        `yaml:"name" json:"name"`
-	Status             pbm.Status    `yaml:"status" json:"status"`
-	Error              *string       `yaml:"error,omitempty" json:"error,omitempty"`
-	LastTransitionTS   int64         `yaml:"-" json:"last_transition_ts"`
-	LastTransitionTime string        `yaml:"last_transition_time" json:"-"`
-	Nodes              []RestoreNode `yaml:"nodes,omitempty" json:"nodes,omitempty"`
+	Name               string        `json:"name" yaml:"name"`
+	Status             pbm.Status    `json:"status" yaml:"status"`
+	Error              *string       `json:"error,omitempty" yaml:"error,omitempty"`
+	LastWriteTS        int64         `json:"last_write_ts" yaml:"-"`
+	LastTransitionTS   int64         `json:"last_transition_ts" yaml:"-"`
+	LastWriteTime      string        `json:"last_write_time" yaml:"last_write_time"`
```

Comment:
Restore doesn't have `LastWriteTime` (no need in L`astWriteTS` for that reason).